package.json

```diff
@@ -80,7 +80,7 @@
     "joi": "^13.4.0",
     "jsonwebtoken": "^8.3.0",
     "jwks-rsa": "^1.3.0",
-    "linkify-it": "^2.0.3",
+    "linkify-it": "^2.1.0",
```

Comment:
We've deprecated use of `linkify-it`, can you remove it from `package.json`?

Reply:
Yes definitely!